README.md

```diff
@@ -22,3 +22,4 @@ The code and information is spread across multiple repositories. The following l
 * [Notification Provider](https://github.com/nextcloud/twofactor_nextcloud_notification)
 * [TOTP Provider](https://github.com/nextcloud/twofactor_totp)
 * [U2F Provider](https://github.com/nextcloud/twofactor_u2f)
```

Comment:
```suggestion
* [U2F Provider](https://github.com/nextcloud/twofactor_u2f)(deprecated)
```

Reply:
not yet. I will add this when we officially deprecate it :+1: